### ISSUE TITLE
Cannot generate queries with custom trained model

### ISSUE THREAD

**Issue by hist0613** [castorini/docTTTTTquery#11]
Problem: my custom T5 model generates **nothing**, resulting in 1M lines of linebreak('\n'). 
- I only fed augmented version of doc_query_pairs.train.tsv file (having 4x many samples) during training. 

**My loss was very small (under 1e-4) when the training is done. Is it correctly trained? Could you share the loss graph? or tell me how small the loss was in your implementation. It would be a great help**

For information, there is a list of minor changes I made. 
- use TPU v2-8 with tf-version 1.15
- use "cut -f2 collection.tsv > input_docs.txt" (cut command uses one-based index)
- reduce the for-loop in shell script into {00..08}, because {00..09} includes "09" for documents id >= 
9M (MSMARCO only has 8.8M documents)
- remove "decode_from_file." for input_filename and output_filename (if not, NotFound error occurred)

**Is there any problematic change? I've changed the decoding method (sample->greedy), but the results haven't changed.**

**Comment by rodrigonogueira4** [comment]
Hi Choi, sorry for the late reply.

The loss seem to be right. Could you share the command you used to train and run eval? Also, one quick experiment is to overfit the model on a very small training set (such as 256 examples only), and see if the model generate the target training queries at inference time.

**Comment by rodrigonogueira4** [comment]
Closing this as there is no activity.
